pkgs/misc/emulators/retroarch/kodi-advanced-launchers.nix

```diff
@@ -1,6 +1,4 @@
-{ stdenv, pkgs, lib, cores, runtimeShell }:
-
-assert cores != [];
```

Comment:
Needed to remove the assertion since it would evaluate the assertion before the override occurs.

But I tested and this works fine too:

```nix
    (kodi-retroarch-advanced-launchers.override {
      cores = with pkgs.libretro; [
        snes9x
        desmume
      ];
    })
```